Dockerfile.caasp-1_0

```diff
@@ -61,7 +64,10 @@ RUN RUBY_VERSION=`rpm --eval '%{rb_default_ruby_abi}'` && \
   yast2-devtools \
   hwinfo-devel \
   yast2-ycp-ui-bindings-devel \
-  && zypper clean -a
+  && zypper clean -a \
+  && rm -rf /usr/lib*/ruby/gems/*/cache/ \
```

Comment:
do we have any gems here for cpp?

Reply:
Yes, we still need at least `yast-rake`.